init_docs/Text.json

```diff
@@ -917,5 +917,7 @@
     "PasswordResetLogin": "Password Reset and logged in",
     "Msg_Re_ResetPassword": "You have recently updated password. Do you want to reset your password again ?",
     "Msg_Success_ResetPassword_ChangePassword": "Your request has been approved. You can change your password.",
-    "Msg_Confirm_ResetPassword_ChangePassword": "You have recently updated password. Do you want to reset your password again ?If you click No then it will resend the password email again."
-}
+    "Msg_Confirm_ResetPassword_ChangePassword": "You have recently updated password. Do you want to reset your password again ?If you click No then it will resend the password email again.",
+    "Nation_Rejected": "Community registration request rejected",
+    "Nation_Rejected": "Community registration request Pending"
```

Comment:
should this not be something other then `Nation_Rejected`
maybe `Nation_Pending` ?